File: 26RemoveDuplicatesSortedArray/RemoveDuplicates.cs
public class Solution {
    public int RemoveDuplicates(int[] nums) {
        int previous = -101;
        int validIndex = 0;
        for(int i = 0; i < nums.Length; i++){
            if(nums[i] != previous){
                nums[validIndex] = nums[i];
                previous = nums[i];
                validIndex++;
            }
        }

        return validIndex;
    }
}

File: 26RemoveDuplicatesSortedArray/removeDuplicates.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int validIndex = 1;
        for (int i = 1; i < nums.length; i++) {
            if (nums[i] != nums[i - 1]) {
                nums[validIndex++] = nums[i];
            }

        }
        return validIndex;
    }
}